config/config.go

```diff
@@ -13,6 +13,12 @@ var (
 	BlockchainInterval  = 3000 * time.Millisecond
 	SystemStatsInterval = 500 * time.Millisecond
 	WidgetInterval      = 1000 * time.Millisecond
+	LinechartInterval   = 5000 * time.Millisecond
+	TimestampEC2        = "2006-01-02T15:04:05.000000000Z"
```

Comment:
TimeStamp format? Is there any package that can the timestamp?

Reply:
Using layout with go time package